src/components/Alert.vue

```diff
@@ -53,7 +54,8 @@ export default {
     flat: Boolean,
     border: String,
     tile: Boolean,
-    dense: Boolean
+    dense: Boolean,
+    'colored-border': Boolean
```

Comment:
Same here coloredBorder (read Vue docs, how to name props https://vuejs.org/v2/guide/components-props.html)

Reply:
fixed, sorry about that